docs/os-ref.md

```diff
@@ -4,615 +4,552 @@ title: aragonOS reference documentation
+### Application capabilities
 
-This allows for many APM registries to exist with different governance models for
-package publishing and releasing new versions. There is an official Aragon curated one,
-aragonpm.eth, which has very strict restrictions of what gets published and
-very high quality standards, that we use for publishing our core components.
+**Fund recovery**
 
-Different APM registries in which everyone can publish their packages are expected to
-be created by the community.
+By default, all AragonApps have a fund recovery mechanism enabled for all tokens and ETH to protect against the event of an accidental transfer of funds. This is partly motivated by a flaw in the ERC20 specification that does not allow contracts to prevent themselves from receiving tokens like they can with ETH.
 
-This diagram tries to illustrate the architecture of an APM Registry DAO:
+All AragonApps expose an externally-accessible fund recovery mechanism:
 
-![](/docs/assets/apm-arch.jpeg)
+```solidity
+function transferToVault(address token) external;
+```
+
+This capability is configurable through the overloadable hook:
+
+```solidity
+function allowRecoverability(address token) public view returns (bool);
+```
 
-### 6.2 APMRegistry
-#### 6.2.1 ENSSubdomainRegistrar
+The default implementation of `allowRecoverability()` is just to return true for all tokens but your overload could choose to not allow certain tokens or even ETH.
 
-Ownership of the ENS name that the `APMRegistry` operates has to be transferred to
-the `ENSSubdomainRegistrar` app that is part of the DAO.
+**Depositable proxies**
 
-The `APMRegistry` needs to have permission on the `ENSSubdomainRegistrar` to create
-new names, which is done every time a repo is created.
+AppProxies start off not being able to receive ETH through the native, gas-limited `.send()` and `.transfer()` methods. This can be explicitly enabled through the `setDepositable()` function when an app wants to allow itself (as the proxy instance) to recieve ETH from other contracts:
 
-#### 6.2.2 APMRegistry governance
+```solidity
+function setDepositable(bool depositable) internal;
+```
 
-Each instance of an APMRegistry can have different governance. Governance of a
-registry is enforced directly using the DAO's ACL.
+An example use case would be a fundraising application which would only enable its proxy instances to be capable of receiving ETH for the duration of a fundraiser.
 
-By default a new repo will set the creator (or `dev` param) as the owner of the
-repo and it is the only address that can create new versions in the repo. However,
-as the permission manager, this account can grant permission to other entities to
-create versions. These entities can be anything from another dev, to a multisig or
-a full blown DAO.
+**Forwarding and EVMScripts**
 
-### 6.3 Repos
+See the following [Forwarders and EVMScripts](#forwarders-and-evmscripts) section.
```

Comment:
I don't think this link is relevant here

Reply:
I think having the link is still useful, since in this context it's not really clear what EVMScripts are for. I've moved the link down to the bottom of the section and reworded it in 7b743b6.